docs/installation/compatibility.md

```diff
@@ -0,0 +1,53 @@
+| AKS (Azure CNI (Legacy)) | 🟢 | `CrossClusterAPIServerInteraction` and `NodePortExposition` |
+| AKS (Kubenet) | 🟢 | `CrossClusterAPIServerInteraction` and `ExternalIPRemapping` |
+| EKS | 🟢 | `CrossClusterAPIServerInteraction` and `ExternalIPRemapping` |
+| GKE (Dataplane v1) | ✅ | No known issues |
+| GKE (Dataplane v2) | 🟢 | `NodePortExposition` and `LoadBalancerExposition` |
+| Aruba Cloud KaaS | ✅ | No known issues |
+| OpenShift | 🔵 | **Work in progress**: all Liqo functionalities except for the networking module work as expected. The team is actively working on adding full networking support. You can still use it by [disabling the Network Module](AdvancedUseOnlyOffloadingDisableModule). |
+| *Your K8s Distribution* | 🟢 | Liqo is designed to work with most Kubernetes-compliant distributions. Your provider is likely supported, but may require specific configurations. |
+
+### Help Us Improve
+
+Have you tested Liqo with a provider not listed here?
+We'd love to hear about your experience!
+Join our [Slack community](https://liqo-io.slack.com/join/shared_invite/zt-h20212gg-g24YvN6MKiD9bacFeqZttQ) and share your test results.
+Your feedback helps us improve Liqo's compatibility across different environments.
+
+## Issues Reference
+
+The following issues are known to affect the compatibility of Liqo with different Kubernetes providers:
+
+- `CrossClusterAPIServerInteraction`: The ability of offloaded pods to properly interact with the Kubernetes API server of the home cluster. This ensures that applications running in remote clusters can still access and manipulate Kubernetes resources (such as ConfigMaps, Secrets, or other custom resources) in their original cluster. Limitations in this area may impact applications that rely on the Kubernetes API for normal operation. See [here](../advanced/kubernetes-api.md) for more details.
+
+- `RemoteExec`: The capability to execute commands in pods that have been offloaded to remote clusters, using `kubectl exec`. This feature is important for debugging, troubleshooting, and administrative tasks on pods running in remote clusters. When limited, administrators may face challenges in directly interacting with offloaded workloads.
+
+- `NodePortExposition`: The ability to make services accessible through NodePort when those services target pods that have been offloaded to remote clusters. This ensures that network traffic addressed to NodePort services in the home cluster is properly routed to the correct pods, even when those pods are running in remote clusters. Limitations here may affect external access to applications.
+
+- `LoadBalancerExposition`: The capability to expose services via LoadBalancer type when those services target pods running in remote clusters. This ensures that cloud provider load balancers can properly distribute traffic to offloaded pods. When this feature has limitations, it may impact the ability to use cloud load balancers with offloaded workloads.
```

Comment:
```suggestion
- `LoadBalancerExposition`: The capability to expose remote offloaded pods via LoadBalancer services. This ensures that cloud provider load balancers can properly distribute traffic to offloaded pods. When this feature has limitations, it may impact the ability to use cloud load balancers with offloaded workloads.
```